cli/cmd/admin/start.go

```diff
@@ -77,6 +77,7 @@ type Config struct {
 	ActivitySinkKafkaBrokers string                 `default:"" split_words:"true"`
 	ActivityUISinkKafkaTopic string                 `default:"" split_words:"true"`
 	MetricsProject           string                 `default:"" split_words:"true"`
+	AutoscalerCron           string                 `default:"" split_words:"true"`
```

Comment:
Change the default to `default:"CRON_TZ=America/Los_Angeles 0 0 * * 1"`

Reply:
Sounds good!